### PR TITLE
doc: remove troubleshooting ses section

### PR DESCRIPTION
**Pull request by greg-fer**

### CODE REVIEW

File: doc/nrf/gs_ins_windows.rst [hunk 1]
```diff
@@ -340,26 +340,57 @@ Complete the following steps to set up the |SES| environment:
...
        :alt: Zephyr Base Not Set prompt
 
        Zephyr Base Not Set prompt
 
+.. buildenv_end
+
+.. buildenv_path_start
+
+4. Make sure the locations of tools are added to the PATH variable.
+On Windows and Linux, SES uses the PATH variable to find executables.
```
Comment (ru-fu) [inline doc/nrf/gs_ins_windows.rst]: Indentation.

File: doc/nrf/gs_ins_windows.rst [hunk 2]
```diff
@@ -340,26 +340,57 @@ Complete the following steps to set up the |SES| environment:
...
+
+.. _build_environment_settings_changes_win:
+
+.. buildenv_settings_changes_start
+
+Changing the SES environment settings
+-------------------------------------
 
-If you want to change these settings later, click :guilabel:`Tools` -> :guilabel:`Options` and select the :guilabel:`nRF Connect` tab (see :ref:`ses_options_figure`).
+If you want to change the SES environment settings later, click :guilabel:`Tools` -> :guilabel:`Options` and select the :guilabel:`nRF Connect` tab, as shown on the following figure from the Windows installation.
```
Comment (ru-fu) [inline doc/nrf/gs_ins_windows.rst]: screenshot, not figure

File: doc/nrf/gs_ins_windows.rst [hunk 3]
```diff
@@ -340,26 +340,57 @@ Complete the following steps to set up the |SES| environment:
...
 
+.. _ses_options_figure:
+
+.. figure:: images/ses_options.png
+     :alt: nRF Connect SDK options in SES on Windows
+
+     nRF Connect SDK options in SES (Windows)
+
+Changing the SES environment settings might require modifying the the PATH variable.
```
Comment (ru-fu) [inline doc/nrf/gs_ins_windows.rst]:
```suggestion
Changing the SES environment settings might require modifying the PATH variable.
```

File: doc/nrf/gs_ins_windows.rst [hunk 4]
```diff
@@ -340,26 +340,57 @@ Complete the following steps to set up the |SES| environment:
...
+If you get an error that a tool or command cannot be found, first make sure that the tool is installed.
+If it is installed, verify its location is correct in the PATH variable.
```
Comment (ru-fu) [inline doc/nrf/gs_ins_windows.rst]:
```suggestion
If it is installed, verify that its location is correct in the PATH variable.
```

File: doc/nrf/gs_ins_windows.rst [hunk 5]
```diff
@@ -340,26 +340,57 @@ Complete the following steps to set up the |SES| environment:
...
 
+.. _ses_options_figure:
+
+.. figure:: images/ses_options.png
+     :alt: nRF Connect SDK options in SES on Windows
+
+     nRF Connect SDK options in SES (Windows)
+
+Changing the SES environment settings might require modifying the the PATH variable.
+If you get an error that a tool or command cannot be found, first make sure that the tool is installed.
```
Comment (ru-fu) [inline doc/nrf/gs_ins_windows.rst]: This is a problem that you won't see during installation, but when you are building for the first time. Does it actually make sense to move this out of the building topic?

File: doc/nrf/gs_programming.rst [hunk 6]
```diff
@@ -73,47 +85,4 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
...
-On macOS, the global PATH variable is used only if you start SES from the command line.
-If you start SES by running the file :file:`bin/emStudio`, the global PATH is not used, and you must specify the path to all executables under **Tools -> Options** (select the **nRF Connect** tab, see :ref:`ses_options_figure`).
-
-In addition, specify the path to the west tool as additional CMake option, replacing *path_to_west* with the path to the west executable (for example, ``/usr/local/bin/west``):
-
-.. parsed-literal::
-   :class: highlight
-
-   -DWEST=\ *path_to_west*
+If you get an error that a tool or command cannot be found, first make sure that the tool is installed.
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst]: ... and then? ;)

File: doc/nrf/gs_ins_mac.rst [hunk 7]
```diff
@@ -64,3 +64,39 @@ Install Homebrew and install the required tools using the ``brew`` command line
...
+
+.. _additional_mac_setup:
+
+Setting up executables on macOS
+===============================
+
+If you start SES on macOS by running the file :file:`bin/emStudio`, make sure to complete the following steps:
+
+1. Specify the path to all executables under :guilabel:`Tools` -> :guilabel:`Options` (in the :guilabel:`nRF Connect` tab).
+   The global PATH on macOS is used only when :ref:`using the command line environment <build_environment_cli_mac>`.
```
Comment (ru-fu) [inline doc/nrf/gs_ins_mac.rst]: No, this is wrong. It is used when starting SES from a command prompt.

File: doc/nrf/gs_ins_mac.rst [hunk 8]
```diff
@@ -64,3 +64,39 @@ Install Homebrew and install the required tools using the ``brew`` command line
...
+
+       -DWEST=\ *path_to_west*
+
+.. _build_environment_cli_mac:
+
+.. include:: gs_ins_windows.rst
+   :start-after: buildenv_cli_start
+   :end-before: buildenv_cli_end
+
+.. note::
```
Comment (ru-fu) [inline doc/nrf/gs_ins_mac.rst]: Wrong location for the note. This section is about building from the command line, which is the alternative to building with SES.

File: doc/nrf/gs_ins_windows.rst (hunk 5 above)
Comment (greg-fer) [reply doc/nrf/gs_ins_windows.rst]: I see. I'll mention it only in building then. Thanks.

File: doc/nrf/gs_programming.rst (hunk 6 above)
Comment (greg-fer) [reply doc/nrf/gs_programming.rst]: And then I moved here the line you commented on in the previous comment ;)

File: doc/nrf/gs_ins_mac.rst (hunk 7 above)
Comment (greg-fer) [reply doc/nrf/gs_ins_mac.rst]: Rephrased.

File: doc/nrf/gs_ins_mac.rst (hunk 8 above)
Comment (greg-fer) [reply doc/nrf/gs_ins_mac.rst]: Moved the section up into the Setting up executables on macOS subsection and rephrased it.

**Comment by greg-fer** [comment]
Hey @ru-fu , do you know what the reason for this error is?
```
/jenkins_workspace/workspace/connect-nrf_documentation_master/_build/nrf/rst/doc/nrf/gs_ins_windows.rst:374: WARNING: duplicate label ses_options_figure, other instance in /jenkins_workspace/workspace/connect-nrf_documentation_master/_build/nrf/rst/doc/nrf/gs_ins_linux.rst
```
The linux file reuses this fragment from the windows file.

**Comment by ru-fu** [comment]
Every label can only occur once. So if you have a label in a section, you cannot reuse that section somewhere else (because it would make the same label occur twice).

File: doc/nrf/gs_ins_mac.rst [hunk 9]
```diff
@@ -64,3 +64,38 @@ Install Homebrew and install the required tools using the ``brew`` command line
...
+
+#. Specify the path to the west tool as additional CMake option, replacing *path_to_west* with the path to the west executable (for example, ``/usr/local/bin/west``):
+
+    .. parsed-literal::
+       :class: highlight
+
+       -DWEST=\ *path_to_west*
+
+
+You can also start SES on macOS from the command line environment.
```
Comment (ru-fu) [inline doc/nrf/gs_ins_mac.rst]: I would replace this sentence and the following with:

If you start SES from the command line, it uses the global PATH variable to find the executables, and you do not need to explicitly configure them.

File: doc/nrf/gs_ins_mac.rst [hunk 10]
```diff
@@ -64,3 +64,38 @@ Install Homebrew and install the required tools using the ``brew`` command line
...
+Unlike on Windows and Linux, this is the only case when macOS uses the global PATH variable to find executables.
+If you get an error that a tool or command cannot be found, first make sure that the tool is installed.
```
Comment (ru-fu) [inline doc/nrf/gs_ins_mac.rst]: I suggest to have this as a separate paragraph and update it to cover both ways of running SES (using the SES configuration works in both cases).
This sentence can stay as is, the next should be something like: 

If it is installed, verify that its path is configured correctly in the SES settings or in the PATH variable.

File: doc/nrf/gs_ins_windows.rst [hunk 11]
```diff
@@ -340,26 +340,59 @@ Complete the following steps to set up the |SES| environment:
...
+.. buildenv_path_end
+
+.. _build_environment_settings_changes_win:
+
+.. buildenv_settings_changes_intro_start
+
+Changing the SES environment settings
+-------------------------------------
+
+If you want to change the SES environment settings later, click :guilabel:`Tools` -> :guilabel:`Options` and select the :guilabel:`nRF Connect` tab, as shown on the following screenshot from the Windows installation.
```
Comment (ru-fu) [inline doc/nrf/gs_ins_windows.rst]: Remove "later"

File: doc/nrf/gs_ins_windows.rst [hunk 12]
```diff
@@ -340,26 +340,59 @@ Complete the following steps to set up the |SES| environment:
...
+.. _ses_options_figure:
+
+.. figure:: images/ses_options.png
+     :alt: nRF Connect SDK options in SES on Windows
+
+     nRF Connect SDK options in SES (Windows)
+
+.. buildenv_settings_changes_ctd_start
+
+Changing the SES environment settings might require modifying the PATH variable.
```
Comment (ru-fu) [inline doc/nrf/gs_ins_windows.rst]: No, these are independent. I *think* (not sure - @tejlmand do you know?) that the nRF Connect SDK options in SES have priority over the PATH. So if you configure a tool in SES, that is the one that is used, no matter what is in the PATH. But you cannot configure all tools in there, that's why we also need the PATH ...
But if you change any configuration in SES, you don't need to change anything in the PATH.

File: doc/nrf/gs_ins_windows.rst [hunk 13]
```diff
@@ -340,26 +340,59 @@ Complete the following steps to set up the |SES| environment:
...
        :alt: Zephyr Base Not Set prompt
 
        Zephyr Base Not Set prompt
 
+.. buildenv_end
+
+.. buildenv_path_start
+
+4. Make sure the locations of tools are added to the PATH variable.
+   On Windows and Linux, SES uses the PATH variable to find executables.
```
Comment (ru-fu) [inline doc/nrf/gs_ins_windows.rst]: Only those executables that are not explicitly configured, see my other comment.

File: doc/nrf/gs_ins_windows.rst (hunk 12 above)
Comment (tejlmand) [reply doc/nrf/gs_ins_windows.rst]: > So if you configure a tool in SES, that is the one that is used, no matter what is in the PATH. 

Correct.

> But you cannot configure all tools in there, that's why we also need the PATH ...

Also correct. Ideally it should be possible to specify any tool in the SES, but as we cannot control if CMake or python suddenly call out for yet another tool, it is only possible to specify those that are common to give issues if not in correct version.
Secondly, being able to specify all tools, could give a too long list.

File: doc/nrf/gs_programming.rst [hunk 14]
```diff
@@ -73,47 +86,5 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
...
-On macOS, the global PATH variable is used only if you start SES from the command line.
-If you start SES by running the file :file:`bin/emStudio`, the global PATH is not used, and you must specify the path to all executables under **Tools -> Options** (select the **nRF Connect** tab, see :ref:`ses_options_figure`).
-
-In addition, specify the path to the west tool as additional CMake option, replacing *path_to_west* with the path to the west executable (for example, ``/usr/local/bin/west``):
-
-.. parsed-literal::
-   :class: highlight
-
-   -DWEST=\ *path_to_west*
+If it is installed, verify that its location is correct in the PATH variable.
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst]:
```suggestion
If it is installed, verify that its location is correct in the PATH variable or, if applicable, in the SES settings.
```

**Comment by ru-fu** [comment]
One more small suggestion, otherwise it looks good. :)

**Pull request merged**